django_reference_app/tests/test_model.py

```diff
@@ -0,0 +1,11 @@
+from django import test
+from django_reference_app.models import Person
+
+class PersonTest(test.TestCase):
+  def create_person(self, name="test", job="only a test"):
+    return Person.objects.create(name=name, job=job)
+
+  def test_person_creation(self):
+    w = self.create_person()
```

Comment:
I think it'd be clearer if the `Person` object was just created inline here rather than via a helper method, and also with a more descriptive variable name:

```
person = Person.objects.create(name='test', job='a test')
```